### PR TITLE
Toonin through extention

### PR DESCRIPTION
**Pull request by alir128**

### CODE REVIEW

File: extension/src/js/background/index.js [hunk 1]
```diff
@@ -8,21 +8,104 @@ const constraints = {
...
+                        console.log("No candidate for RTC connection");
+                        return;
+                    }
+                    socket.emit("peer new ice", {
+                        id: socket.id,
+                        room: room,
+                        candidate: event.candidate
+                    });
+                };
+                rtcConn3.onaddstream = event => {
```
Comment (ArsalaBangash) [inline extension/src/js/background/index.js]: onAddStream is being deprecated https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/onaddstream

Comment (ArsalaBangash) [reply extension/src/js/background/index.js]: We are now using onAddTrack

File: extension/src/manifest.json [hunk 2]
```diff
@@ -7,13 +7,14 @@
...
     },
-    "web_accessible_resources": [
-        "js/inject.js"
-    ],
     "background": {
         "scripts": ["js/lib/adapter.js", "js/lib/socket.io.js", "js/background.js"],
         "persistent": true
     },
+    "content_scripts": [{
+        "matches": ["<all_urls>"],
```
Comment (ArsalaBangash) [inline extension/src/manifest.json]: What is the purpose of this?

File: extension/src/js/background/index.js [hunk 3]
```diff
@@ -64,21 +152,20 @@ function loadSocketIO() {
...
 function injectAppScript() {
-    chrome.tabs.executeScript({
-        file: "js/inject.js"
-    }, function () {
-        if (chrome.runtime.lastError) {
-            console.error(chrome.runtime.lastError.message);
-        } else console.log("All scripts successfully loaded");
-    });
+    chrome.tabs.query({active: true, currentWindow: true}, function(tabs) {
+        var activeTab = tabs[0];
```
Comment (ArsalaBangash) [inline extension/src/js/background/index.js]: We should probably match this with the current Tab ID. Lakshaya stores it in his PR, so maybe we can revisit this change and incorporate that.

File: extension/src/js/background/index.js [hunk 4]
```diff
@@ -8,21 +8,104 @@ const constraints = {
 var tabID;
 var port;
 
+var play = false;
+var firstTime = true;
+
+var room;
+var rtcConn2 = null;
```
Comment (ArsalaBangash) [inline extension/src/js/background/index.js]: rtcConn2 and rtcConn3 could be renamed to something more descriptive

File: extension/src/manifest.json (hunk 2 above)
Comment (alir128) [reply extension/src/manifest.json]: https://stackoverflow.com/questions/16096482/what-does-http-https-and-all-urls-mean-in-the-context-of-ch

**Pull request merged**